dockstore-common/pom.xml

```diff
@@ -107,6 +107,21 @@
             <groupId>io.cwl</groupId>
             <artifactId>cwlavro-tools</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.amazonaws</groupId>
+            <artifactId>aws-java-sdk-core</artifactId>
+            <version>1.10.29</version>
+        </dependency>
+        <dependency>
+            <groupId>com.amazonaws</groupId>
```

Comment:
sdk-core and sdk-s3 should have their versions specified in the root pom.xml already